tsl/test/expected/compression.out

```diff
@@ -805,3 +805,21 @@ EXPLAIN (costs off) SELECT * FROM test_collation WHERE val_2 < 'a' COLLATE "C";
          Filter: (val_2 < 'a'::text COLLATE "C")
 (23 rows)
 
+--test datatypes
+CREATE TABLE datatype_test(time timestamptz NOT NULL, i2 int2, i4 int4, i8 int8, f4 float4, f8 float8, d date, ts timestamp, tstz timestamptz, i interval, n numeric, d2 decimal, t text, c char);
+SELECT create_hypertable('datatype_test','time');
+      create_hypertable      
+-----------------------------
+ (11,public,datatype_test,t)
+(1 row)
+
+ALTER TABLE datatype_test SET (timescaledb.compress);
+INSERT INTO datatype_test VALUES ('2000-01-01',2,4,8,4.0,8.0,'2000-01-01','2001-01-01 12:00','2001-01-01 6:00','1 week', 3.41, 4.2, 'text', 'x');
```

Comment:
could you add a query to the test case to:
 select from _timescaledb_catalog.hypertable_compression and join with _timescaledb_catalog.compression_algorithm to display the algorithms being used for each column